Cargo.toml

```diff
@@ -137,7 +137,7 @@ rand = "0.8"
 rand_chacha = "0.3.1"
 ratatui = "0.26"
 rayon = "1.10"
-reedline = "0.34.0"
```

Comment:
If you use a [git dependency pointing to your fork and branch](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#choice-of-commit), things can be tested both in CI and more easily by folks just fetching for a functional shakedown.

In general we tend to specify our non-crates-io dependencies only in the [patch section](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#overriding-dependencies) at the bottom of the main `Cargo.toml` (to centralize those dependencies that we can't merge or publish like that, and as the workspace dependency only became a thing post 1.64)

Reply:
Yeah, I wouldn't spend too much time on this and clean it up completely when/if reedline is merged. Then I would only need to  bump its version.